src/ray/raylet/worker_pool.cc

```diff
@@ -1644,10 +1646,16 @@ void WorkerPool::GetOrCreateRuntimeEnv(
       [job_id,
        serialized_runtime_env = std::move(serialized_runtime_env),
        runtime_env_config = std::move(runtime_env_config),
+       start,
        callback](bool successful,
                  const std::string &serialized_runtime_env_context,
                  const std::string &setup_error_message) {
         if (successful) {
+          auto end = std::chrono::high_resolution_clock::now();
+          auto duration =
+              std::chrono::duration_cast<std::chrono::milliseconds>(end - start);
+          stats::RuntimeEnvCreationTimeMs.Record(duration.count());
```

Comment:
Should we measure the worker startup time too?

Reply:
is this old way of recording metrics?

Reply:
is `ProcessStartupTimeMs` what you are referring to?